extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -494,12 +494,16 @@
 			_t.reset();
```

Comment:
[2] ...but could you please implement another way to suppress the useless call to `reset()`?